docs/device-manager.md

```diff
@@ -0,0 +1,40 @@
+PingViewer automatically scans for available devices on serial ports (COM) and UDP connections. If a Ping device is detected, the available connection is shown and the user will be able to choose between all available devices in **Device Manager**.
+
+PingViewer attempts to connect to UDP port 9090 [Ping1D] and 9092 [Ping360] on the host at IP address 192.168.2.2. This is the default IP address of the **Companion computer** that runs on the BlueROV2. The **Companion computer** automatically configures a communication bridge on these ports to a connected sensor.
+
+![Device Manager](/ping-viewer/images/viewer/device-manager-1.png)
+
+    1- Shows an animation and the name of the available sensor.
+    2- Describes the kind of the available connection.
+    3- Shows the availability of the connection.
+        **Red**: The connection is not available/reliable.
+        **Green**: The sensor is connected and available for connection.
+    4- Describes the connection type and the connection configuration.
+    5- Allows the user to go to **Manual Connection**.
+    6- Cancel and exit **Device Manager**.
+
```

Comment:
I think "manual connection" should be a header too (## Manual Connection).
And maybe "connection types" not, or make it a sub-header